lib/ae_mdw/db/model.ex

```diff
@@ -743,97 +676,28 @@ defmodule AeMdw.Db.Model do
   def record(AeMdw.Db.Model.IntTransferTx), do: :int_transfer_tx
   def record(AeMdw.Db.Model.KindIntTransferTx), do: :kind_int_transfer_tx
   def record(AeMdw.Db.Model.TargetKindIntTransferTx), do: :target_kind_int_transfer_tx
-
   def record(AeMdw.Db.Model.DeltaStat), do: :delta_stat
   def record(AeMdw.Db.Model.TotalStat), do: :total_stat
+  def record(AeMdw.Db.Model.None), do: :none
+
+  #
+  # Stub for Node-Mdw database management
+  #
+  @none_defaults [index: 0, value: 0]
+  defrecord :none, @none_defaults
+
+  @spec tables() :: list(atom())
+  def tables(), do: [AeMdw.Db.Model.None]
+
+  @spec records() :: list(atom())
+  def records(), do: [:none]
```

Comment:
Why do we need these?

Reply:
The behaviour for this Node database integration should be better analyzed later. For now leaving just a dummy table for the `AeMdw.Db.Setup`.